src/templates/docPage.js

```diff
@@ -109,12 +107,6 @@ const BasicDoc = ({ data, location, pageContext }) => {
             />
           )}
           <TableOfContents headings={headings} />
-          <RelatedResources
```

Comment:
what happens if a content author adds related resources via frontmatter?

Reply:
they become very saddened when it doesn't show up... 😂 just reverted that